spec/acceptance/reboot_resume_spec.rb

```diff
@@ -59,10 +59,10 @@ def apply_reboot_manifest(agent, reboot_manifest, match)
       MANIFEST
 
     posix_agents.each do |agent|
-      apply_manifest_on agent, remove_artifacts
+      execute_manifest_on(agent, remove_artifacts)
```

Comment:
Note that this is in `after(:all)` section and only applies to agents.  This is safe, and probably preferred, to a master-agent workflow